src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -254,17 +260,27 @@ public override OneOrMany<SyntaxList<AttributeListSyntax>> GetAttributeDeclarati
                             throw ExceptionUtilities.UnexpectedValue(accessor.Kind());
                     }
 
-                    if (accessor.Body != null || accessor.ExpressionBody != null)
+                    var body = (SyntaxNode?)accessor.Body ?? accessor.ExpressionBody;
+                    if (body != null)
                     {
                         accessorsHaveImplementation = true;
                     }
+
+                    usesFieldKeyword = usesFieldKeyword || containsFieldKeyword(accessor);
```

Comment:
The other usage of this local function passes the `body` as an argument, should we do the same here?

Reply:
The reason for passing `accessor` rather than `body` is to include attributes from the accessor.